### ISSUE TITLE
ValueError: invalid literal for int() with base 10: '-'

### ISSUE THREAD

**Issue by ProfessorVenkman** [Electron-Cash/Electron-Cash#1757]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
  File "/home/ago/src/Electron-Cash/gui/qt/network_dialog.py", line 756, in set_tor_socks_port
    socks_port = int(self.tor_socks_port.text())

ValueError: invalid literal for int() with base 10: '-'
```

Reporter
------------

This issue was reported by 1 user(s):

| Electron Cash Version  | Python Version | Operating System  | Wallet Type  | Locale |
|---|---|---|---|---|
| 4.0.12  | 3.8.1 (default, Dec 21 2019, 20:57:38)  [GCC 9.2.0] | Linux-5.4.7-arch1-1-x86_64-with-glibc2.2.5 | standard | de_DE | 

Additional Information
------------------------

The reporting user(s) did not provide additional information.

**Comment by cculianu** [comment]
@EchterAgo was this you?

**Comment by EchterAgo** [comment]
Oh, right, forgot that one.

**Comment by EchterAgo** [comment]
Fixed in e611adacd4394fdeb34a642b0afd105bd1e7bfda

**Comment by EchterAgo** [comment]
@cculianu this can be closed
